extensions/webjars-locator/deployment/src/test/java/io/quarkus/webjar/locator/test/WebJarLocatorRootPathTest.java

```diff
@@ -0,0 +1,66 @@
+package io.quarkus.webjar.locator.test;
+
+import static org.hamcrest.core.Is.is;
+
+import java.util.Arrays;
+
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.bootstrap.model.AppArtifact;
+import io.quarkus.test.QuarkusUnitTest;
+import io.restassured.RestAssured;
+
+public class WebJarLocatorRootPathTest extends WebJarLocatorTestSupport {
+    private static final String META_INF_RESOURCES = "META-INF/resources/";
+
+    @RegisterExtension
+    static QuarkusUnitTest runner = new QuarkusUnitTest()
+            .withApplicationRoot((jar) -> jar
+                    .addAsResource(new StringAsset("<html>Hello!<html>"), META_INF_RESOURCES + "/index.html")
+                    .addAsResource(new StringAsset("Test"), META_INF_RESOURCES + "/some/path/test.txt"))
+            .overrideConfigKey("quarkus.http.root-path", "/app")
```

Comment:
If you configure the root as `/app` is it still expected `RestAssured.get("/")` will get the `index.html` content?

Reply:
My understanding is that, unless we override `RestAssured.basePath` (as [in this case](https://github.com/eliasnorrby/quarkus/blob/8b864470d458c006bf726890b069b01fd5177670/extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/src/test/java/io/quarkus/resteasy/reactive/server/test/path/RestApplicationPathTestCase.java#L42), for example), `RestAssured.get` will take the configured root path into consideration. So a `RestAssured.get("/")` translates to a `GET /app/`, which is expected to return the `index.html`.

If this was not the case, I would expect all the get requests in the test to result in `404`s as none of them specify the `/app` root path prefix.

Reply:
Thanks, looks like I messed something up in my branch, it fails on this test now.